File: Day-06/jagadeeshwaran.py
class Solution:
    def singleNumber(self, nums):
        self.nums = nums
        result  = 0
        for i in range(len(self.nums)):
            result = result ^ self.nums[i]
        return result
if __name__ == '__main__':
    obb = Solution()
    nums = list(map(int, input().strip(' []').split(',')))
    print(obb.singleNumber(nums))

File: Day-83/jagadeeshwaran.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i = 0;
         if(nums.size() == 0)
            {
                return(i);
            }
        for(int j = 0 ; j < nums.size() ; j++)
        {
            if(nums[i] != nums[j])
            {
                i++;
                nums[i] = nums[j];
            }
        }
        return(i+1);
        
    }
};
